scalafix/rules/src/main/scala/fix/v0_14_0/FixAvroCoder.scala

```diff
@@ -145,11 +176,18 @@ class FixAvroCoder extends SemanticRule("FixAvroCoder") {
           // Coder[T] is a variable type where T is an avro type
           findMatchingValTypeParams(t, CoderMatcher)
             .exists(isAvroType)
-        case q"$jobTestBuilder(..$args)" if JobTestBuilderMatcher.matches(jobTestBuilder) =>
+        case q"$fn(..$args)" if JobTestBuilderMatcher.matches(fn) =>
           args.headOption match {
             case Some(q"$io[$tpe](..$args)") if isAvroType(tpe.symbol) => true
             case _                                                     => false
           }
+        case q"$fn(..$args)" if ParallelizeMatcher.matches(fn) =>
```

Comment:
maybe we shouldn't match class/methods, but set `usesAvroCoders = true` if any there is any implicit Coder param?

Reply:
this one is difficult to implement in a generic way 😢 Say you have a method like `def someMethod[T: Coder])(foo: String, bar: Int, baz: T)` that's invoked as `someMethod("foo", 1, newT)`... you basically have to examine every argument to search for one matching T, then analyze its signature

But the number of cases here is growing so much that the overall complexity might not be that much higher, lol. I'll try it out...